packages/salesforcedx-vscode-core/src/commands/forceApexLogGet.ts

```diff
@@ -131,6 +132,9 @@ export class LogFileSelector
         return {
           id: logInfo.Id,
           label: icon + logInfo.LogUser.Name + ' - ' + logInfo.Operation,
+          startTime: moment(new Date(logInfo.StartTime)).format(
```

Comment:
Does adding this cause the startTime to now appear twice in the quickpick dropdown?

Reply:
I had the same question as well

Reply:
No, this is because the quickpick dropdown only prints the info in the default fields of the QuickPickItem interface and ignores the ones we add to it when we extend it as part of `ApexDebugLogItem`.

![screen shot 2019-01-17 at 9 52 15 am](https://user-images.githubusercontent.com/1041842/51338102-cc094e80-1a3d-11e9-8a96-988038304fce.png)

Reply:
Gotcha, thanks. LGTM 👍